### PR TITLE
Creating the foundation for boards and posts

### PR DESCRIPTION
**Pull request by ohboyotero**
Sorry, I let this get too big. Should have broken it into multiple smaller PRs.

This PR does the following:

1. Creates the `Board` and `Post` models.
2. Sets up admin pages for both of them.
3. Creates some shitty placeholder UI for boards and posts on the website
4. Sets up author one-to-many relationship on Posts (one author can author many posts) with the actual Character model (though that happens through `objects.ObjectDB` #becauseEvennia #goals #blessed #python #python3 #pythonstars #madeWithAI).
5. Sets up many-to-many subscription relationships on Board with Characters (also through `objects.ObjectDB`).
6. Migrations for the board and post models.

Still to do:

1. Anything that actually USES this. Today you can only create boards/posts in admin console, and can only read them in admin console or in the crappy placeholder UI in the website.
2. Any in-game commands to interact with this stuff
3. Anything having to do with actually tracking subscriptions, post reads, notifications, etc.

### CODE REVIEW

File: server/conf/settings.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "django.contrib.humanize",
     "web.template_overrides",
     "world.arts",
+    "world.boards",
```
Comment (ohboyotero) [inline server/conf/settings.py:45]: YO, you misspelled `INSTALLE_APPS` you dummy

File: world/scenes/models.py [hunk 2]
```diff
@@ -70,9 +70,6 @@ def addLogEntry(self, type, text_content, character):
             character=character
         )
 
-    def web_get_detail_url(self):
```
Comment (ohboyotero) [inline world/scenes/models.py:73]: Glad that you're making this change, but for fucks sake don't add yet another thing to the same PR. Separate PRs please noooob.

File: world/boards/apps.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class BoardsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```
Comment (ohboyotero) [inline world/boards/apps.py]: Get rid of this. No need for this and it's only going to drift from the defaults in Django over time.

File: world/boards/admin.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from world.boards.models import Board, Post
+
+admin.site.register(Board)
+admin.site.register(Post)
```
Comment (ohboyotero) [inline world/boards/admin.py]: Need a new line at the end of the file to avoid breaking all the things.

Comment (ohboyotero) [reply world/boards/admin.py]:
```python
admin.site.registerForTheParty(Boards)
```
should be used instead

File: world/boards/migrations/0001_initial.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('objects', '0012_character_defaultcharacter_defaultexit_defaultobject_defaultroom_object'),
```
Comment (ohboyotero) [inline world/boards/migrations/0001_initial.py]:
```suggestion
```

File: world/boards/tests.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (ohboyotero) [inline world/boards/tests.py:3]: Very strong test coverage 😗 👌

Comment (ohboyotero) [reply world/boards/tests.py:3]: Smart ass

File: world/boards/admin.py (hunk 4 above)
Comment (ohboyotero) [reply world/boards/admin.py]: Done!

File: world/boards/migrations/0001_initial.py (hunk 5 above)
Comment (ohboyotero) [reply world/boards/migrations/0001_initial.py]: Done!

File: world/boards/apps.py (hunk 3 above)
Comment (ohboyotero) [reply world/boards/apps.py]: Done! ✅

File: world/scenes/models.py (hunk 2 above)
Comment (ohboyotero) [reply world/scenes/models.py:73]: Yeah sorry 😬 will do in the future.

File: server/conf/settings.py (hunk 1 above)
Comment (ohboyotero) [reply server/conf/settings.py:45]: So I did, but I fix it in future PRs so let's let future Daniel solve that problem. BAIII

**Review by dancerdevin — APPROVED** [review]
I love you, bro.

**Pull request merged**